index.html

```diff
@@ -235,20 +235,20 @@ <h3>Latest news</h3>
           <!-- News box Start -->
           <div class="news-box">
             <div class="news-box-title">          
-              <span>25/09/2024</span>
+              <span>15/10/2024</span>
             </div>
             <div class="news-box-content">
-              <p>A new <a href="https://calendar.opensuse.org/teams/uyuni" target="_blank">calendar</a> is available in order to track all events related to Uyuni, Uyuni Community Hours included. Feel free to add it as Webcal, Google Calendar or iCalendar URL!</p>
+	      <p>Uyuni 2024.10 is now available with support for Ubuntu 24.04, compressed POS image templates and new ISO-8601 date format for API endpoints! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```

Comment:
If we need to pick things, I'd rather mention what we're going containers only rather than the compressed POS images or even the change on the API andpoints.